setup.py

```diff
@@ -40,7 +40,8 @@
     install_requires = [
         'ait-core>=1.0.0',
         'pyasn1',
-        'enum34==1.1.6'
+        'enum34==1.1.6',
```

Comment:
We don't need enum34 since we're updating to 3.x. Let's drop that.

The bitstring usage in the code seems unnecessary, especially since we're immediately converting stuff into a bytearray after. Let's drop that and update the code as necessary.